docs/modules/ROOT/pages/api/test_utilities.adoc

```diff
@@ -0,0 +1,317 @@
+= Test Utilities
+
```

Comment:
Since this package is not part of the `openzeppelin` package (`openzeppelin::testing` won't work if users add openzeppelin as a dependency), I think it would be helpful for users if we add a note here at the beginning mentioning how to add it to the Scarb toml.

Reply:
Added a note